projects/news_feed/PSL_Matches.py

```diff
@@ -22,22 +25,33 @@ def main():
     """
         main function
     """
-    all_matches_link = list()
 
-    # get psl matches links
-    psl_matches_links = utilities.grab_psl_matches_urls("match-highlight border-bottom", "lxml")
+    # calling function to grab match number, venue and date
+    match_number, match_venue, match_date = utilities.grab_match_number_date_venue("small mb-0 match-description")
+
+    # calling function to grab the team names
+    team_a, team_b = utilities.grab_team_names('row no-gutters', 'name')
+
+    # calling function to grab the team scores
+    team_a_score, team_b_score = utilities.grab_team_scores('row no-gutters', 'score')
 
-    if not psl_matches_links:
-        raise RuntimeError('No PSL matches link found')
+    # calling function to grab the name of winning team
+    winning_team = utilities.grab_winning_team('extra-small mb-0 match-description match-description-bottom')
 
-        # creating a data frame using pandas library
-    for links in zip(psl_matches_links):
-        all_matches_link.append({'Links For Matches': links})
+    # rearranging all fetched data into tabular form
+    all_fetched_information = list()
+    for match_no, match_Date, match_place, Team1, Team2, \
+        Team1_score, Team2_score, winner \
```

Comment:
the local variables (variables inside functions) shall be in small cases

`Team1` --> `team_1`

make same changes for all other places

Reply:
Firstly, these variables are not a part of function. They have been used temporarily in a for loop for creating pandas list. However, I have changed them as described by you except one in order to avoid conflict

Reply:
@Raspberry-beans its a general python rule.

all variables inside function shall be of small letters and in snake casing e.g. my_name etc.

Reply:
@M-Shan All variables inside my functions are in small letters now. Particularly talking about utilities.py file